pkg/sidecar/k8s_reactor.go

```diff
@@ -221,68 +219,11 @@ func (d *K8sReactor) manageContainer(ctx context.Context, container *docker.Cont
 		return nil, fmt.Errorf("failed to get link by name %s: %w", controlNetworkIfname, err)
 	}
 
-	var servicesIPs []net.IP
-
-	for _, s := range d.allowedServices {
-		// Get the routes to redis, influxdb, etc... We need to keep these.
-		r, err := getServiceRoute(netlinkHandle, s.IP)
-		if err != nil {
-			return nil, fmt.Errorf("cant get route to %s ; %s: %s", s.IP, s.Name, err)
-		}
-		logging.S().Debugw("got service route", "route.Src", r.Src, "route.Dst", r.Dst, "gw", r.Gw.String(), "container", container.ID)
-
-		servicesIPs = append(servicesIPs, r.Dst.IP)
-	}
```

Comment:
In our opinion, it was never necessary to delete them in the first place. The control network was and remains the basic network that is not modified during tests in any way. As we have concluded, the workers use only the data network (and you can confirm this using `tcpdump` to track individual packets) to exchange packets.

We have tested this behavior and have confirmed that skipping route deletion has no impact on the test pods; we have found no valid reason to delete the control routes.

Reply:
>What is a worker?

I meant a test instance pod - we sometimes use those interchangeably. 

>Could you share your testing procedure and the results? (logs, tcpdump, etc)?

Each package is routed using the origin/destination address; since they belong to the data network, the packages do not come in contact with the control network in any way. 
Just to be clear, you wish for us to show that the package is routed correctly through the data network, correct?


> Docs: "After the sidecar  is finished initializing the network, it should be impossible to use the control network to communicate with other test plan instances." 
Is this the only piece of code that does exactly that?

Reply:
That sounds good, thanks for the precision.

Reply:
Do we have a test to check that the instance has access to the control network even if we drop the connection?